apps/store/src/blocks/RichTextBlock/RichTextBlock.tsx

```diff
@@ -14,9 +15,13 @@ export type RichTextBlockProps = SbBaseBlockProps<{
   largeText?: boolean
 }>
 
-export const RichTextBlock = ({ blok }: RichTextBlockProps) => {
+export const RichTextBlock = ({ blok, nested }: RichTextBlockProps) => {
   const content = useMemo(() => renderRichText(blok.content), [blok.content])
 
+  if (nested) {
+    return <NestedRichText largeText={blok.largeText}>{content}</NestedRichText>
+  }
+
```

Comment:
We could us the `ConditionalWrapper` for this as we do it other places where we check if it's a nested block or not :) 

```suggestion
   <ConditionalWrapper
      condition={!nested}
      wrapWith={(children) => (
        <GridLayout.Root>
          <GridLayout.Content
            width={blok.layout?.widths ?? { base: '1', md: '2/3', xl: '1/2' }}
            align={blok.layout?.alignment ?? 'center'}
          >
            {children}
          </GridLayout.Content>
        </GridLayout.Root>
      )}
```